CHANGELOG.md

```diff
@@ -23,8 +23,10 @@ All notable changes to this project will be documented in this file.
 ## [Unreleased]
 * ### ALL
     * #### Added
+        * Support for Python 3.8
     * #### Changed
     * #### Removed
+        * Support for Python 2.7 and 3.4
```

Comment:
Add something like: These versions of Python are no longer supported by the Python Software Foundation, and a link to the support doc.

Reply:
Updated